tests/integ/jobmanager/common.go

```diff
@@ -481,3 +483,7 @@ func (suite *TestJobManagerSuite) TestTestNull() {
 	_, err := suite.startJob(jobDescriptorNullTest)
 	require.Error(suite.T(), err)
 }
+
+func requireErrorType(t *testing.T, err, errType error) {
+	require.True(t, errors.As(err, &errType))
+}
```

Comment:
This function was needed only to separate implementation for different versions of Go. Since there's no support of pre-Go1.13 anymore, this code could be used inline/inplace instead of using this interlayer.